algorand.cpp

```diff
@@ -175,8 +175,26 @@ bool is_present(Address a) {
 bool is_present(LogicSig lsig) {
   return is_present(lsig.logic);
 };
+bool is_present(Subsig subsig) {
+  return is_present(subsig.public_key);
+};
 bool is_present(MultiSig msig) {
-  return msig.threshold > 0;
+  if (msig.threshold == 0) {
```

Comment:
@jannotti I updated this method to only return true if all of the threshold signatures are present based on the previous comment about testing whether or not all threshold signatures public keys and signatures are present. This modifies the behavior slightly now so that the MSig does not get packed if it doesn't meet the threshold which means the node returns an "empty txgroup" error rather than returning a validation failed error on a packed Msig Txn. Does this track with your intent?